datasources/src/test/scala/com/normation/plugins/datasources/UpdateHttpDatasetTest.scala

```diff
@@ -1224,13 +1224,15 @@ class UpdateHttpDatasetTest extends Specification with BoxSpecMatcher with Logga
         nodeIds.map(n => if (expectMod) NodeUpdateResult.Updated(n) else NodeUpdateResult.Unchanged(n): NodeUpdateResult)
       )
 
-      res.either.runNow must beRight(matcher) and (
-        if (expectMod) {
-          updates.get.runNow must havePairs(nodeIds.map(x => (x, 1)).toSeq*)
-        } else {
-          updates.get.runNow must beEmpty
```

Comment:
In previous branches (8.1 and before), this line is `true must _== true`... I wonder if we shouldn't worry about this and force it like that :sweat_smile: